templates/etc/init.d/rpm/td-agent

```diff
@@ -184,6 +184,13 @@ do_start() {
   # Set Max number of file descriptors for the safety sake
   # see http://docs.fluentd.org/en/articles/before-install
   ulimit -n 65536 1>/dev/null 2>&1 || true
+
+  GOOGLE_LOGGING_ENABLE=$(curl --silent -f -H "Metadata-Flavor: Google" http://169.254.169.254/computeMetadata/v1/project/attributes/google-logging-enable 2>/dev/null)
+  if [ -n "$GOOGLE_LOGGING_ENABLE" -a "$GOOGLE_LOGGING_ENABLE" -eq 0 ]; then
+      echo "Disabled via metadata"
+      return 0
```

Comment:
I see the return value isn't documented here as it is for debian, is 0 what we want?  Can you add a comment.